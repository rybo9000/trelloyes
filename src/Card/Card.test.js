import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Card from './Card';

describe('Card Component Tests', () => {
    it('Renders Without Crashing', () => {
        const testDiv = document.createElement('div');
        ReactDOM.render(<Card />, testDiv);
        ReactDOM.unmountComponentAtNode(testDiv);
    })

    it('Renders As Expected', () => {
        const tree = renderer.create(<Card />).toJSON();
        expect(tree).toMatchSnapshot();
    })
})