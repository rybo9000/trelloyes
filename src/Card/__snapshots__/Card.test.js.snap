// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Component Tests Renders As Expected 1`] = `
<div
  className="Card"
>
  <h3 />
  <p />
</div>
`;
